tests/build.rs

```diff
@@ -329,21 +279,14 @@ fn it_builds_with_webpack_target_webworker() {
     "#,
     );
 
-    fixture.create_wrangler_toml(
-        r#"
-        type = "webpack"
-        webpack_config = "webpack.config.js"
-    "#,
-    );
+    let wrangler_toml = WranglerToml::webpack_std_config("test-build-webpack-target-webworker");
+    fixture.create_wrangler_toml(wrangler_toml);
 
     build_creates_assets(&fixture, vec!["script.js"]);
-    fixture.cleanup()
 }
 
 fn build_creates_assets(fixture: &Fixture, script_names: Vec<&str>) {
-    // Lock to avoid having concurrent builds
-    let _g = BUILD_LOCK.lock().unwrap();
-
+    let _lock = fixture.lock();
```

Comment:
calling .lock on the fixture instance rather than the "global" static shared instance seems to have solved some of the domino effect on failing tests.

Reply:
nice :)